### PR TITLE
transaction wording update for SDK API 3.3

### PR DESCRIPTION
**Pull request by osfameron**
see https://github.com/couchbase/docs-sdk-common/pull/49

### CODE REVIEW

File: modules/concept-docs/pages/durability-replication-failure-considerations.adoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ include::7.0@sdk:shared:partial$durability-replication-failure-considerations.ad
 include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=performance]
 
 include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=txns]
+* xref:project-docs:distributed-transactions-java-release-notes.adoc[Transactions Release Notes].
```
Comment (RichardSmedley) [inline modules/concept-docs/pages/durability-replication-failure-considerations.adoc:24]: Is this in the right place?
Should it be in the Further Reading section?

Comment (osfameron) [reply modules/concept-docs/pages/durability-replication-failure-considerations.adoc:24]: It was originally in the linked `tag=txns` partial, so this refactor leaves it in the same bulleted list.

Comment (osfameron) [reply modules/concept-docs/pages/durability-replication-failure-considerations.adoc:24]: (intent is to be able to link the same partial from e.g. PHP sdk, without any Java specifics)

Comment (RichardSmedley) [reply modules/concept-docs/pages/durability-replication-failure-considerations.adoc:24]: My digging into the changes without building got a little tangled  :)

Comment (maria-robobug) [reply modules/concept-docs/pages/durability-replication-failure-considerations.adoc:24]: Makes sense to me, unless we think this wasn't the right place for it to begin with?

**Review by maria-robobug — APPROVED** [review]
👍

**Pull request merged**